UPGRADE.md

```diff
@@ -1,5 +1,15 @@
 # Upgrade
 
+## dev-master
+
+### Filter
+
+Update the schema `app/console doctrine:schema:update --force` and run following SQL-Statement:
+ 
+```sql
+UPDATE re_conditions SET value = CONCAT('"', value, '"') WHERE value NOT LIKE '"%"'
```

Comment:
@webmozart Was your question why we are not using doctrine migrations, or why we don't want to introduce them now?

I can't think of any good reason not to use doctrine migrations, but introducing a new dependency in a hotfix is a bit weird, isn't it? Although this entire feature should not be a hotfix...

So I don't have a strong opinion here, maybe we can add the new dependency, since this should actually be considered as a new feature anyway.